### ISSUE TITLE
the output score of the keypoint

### ISSUE THREAD

**Issue by wxhqw** [Gengzigang/PCT#13]
the output score of the keypoint more than 1, what does the output score means

**Comment by Gengzigang** [comment]
You can see that we adopt the maximum classification logits values as our scores in this line of code: https://github.com/Gengzigang/PCT/blob/main/models/pct_head.py#L153. You can also use the logits after passing through softmax as scores, which will not have any impact on performance.

**Comment by wxhqw** [comment]
> You can see that we adopt the maximum classification logits values as our scores in this line of code: https://github.com/Gengzigang/PCT/blob/main/models/pct_head.py#L153. You can also use the logits after passing through softmax as scores, which will not have any impact on performance.

thanks
